CMakeLists.txt

```diff
@@ -109,7 +110,7 @@ if(IOS AND IOS_SDK_VERSION LESS 10.0)
   ament_export_libraries(pthread)
 endif()
 
-install(TARGETS ${PROJECT_NAME}
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}
```

Comment:
The template in `ros2cli` calls this `export_${PROJECT_NAME}`. Is there a collision we should be concerned about here? Or maybe it would just be a good idea to align with the template. Just a thought.

Reply:
No, there is no collision. Target names and export names live in separate namespaces. I don't know why @Karsten1987 used that prefix in the package template. Maybe to avoid users getting confused what `${PROJECT_NAME}` is used for in both cases.